de.bund.bfr.knime.fsklab.nodes/src/de/bund/bfr/knime/fsklab/v2_0/editor/FSKEditorJSNodeModel.java

```diff
@@ -402,7 +403,7 @@ public void saveCurrentValue(NodeSettingsWO content) {
                   ConversionUtils.ModelClass.valueOf(m_config.getModelType()));
             //update representation metadata
             viewRep.setModelMetadata(MAPPER.writeValueAsString(metadata));
-            originalMetadata = MAPPER.readValue(viewRep.getModelMetadata(), modelClass);
```

Comment:
yes, it's the reason for the https://github.com/RakipInitiative/ModelRepository/issues/356 .
the originalMetadata should be reset if a conversion happens because it used only to find out if there is any changes happen to the metadata like add, deleting or editing params which is not the case in conversion usecase.